mitmproxy/proxy/server.py

```diff
@@ -506,6 +506,12 @@ async def handle_hook(self, hook: commands.StartHook) -> None:
 
     opts = moptions.Options()
     # options duplicated here to simplify testing setup
+    opts.add_option(
+        "store_streamed_bodies",
+        bool,
+        False,
+        "Store HTTP request and response bodies when streamed.",
```

Comment:
This is for testing only. Let's do empty string for all help texts to make it clear that these are not used anywhere in practice.

```suggestion
        "",
```

Reply:
Done in the latest commit